.gitignore

```diff
@@ -42,3 +42,6 @@ __TMP
 .metals/
 __azurite_*
 /.idea
+
```

Comment:
Can be removed?

Reply:
I wanted to bring this up as well. By default, all .terraform.lock.hcl files are being pushed, but do we want it that way? This would include files generated locally in `modules`, such as those in `common`.